File: seminar1/task1.py
import random

def find_imperative(array, target):
    for num in array:
        if num == target:
            return True
    return False

array = []
for i in range(10):
    array.append(random.randint(-10, 10))
print(array)

print(find_imperative(array, 5))

File: seminar1/Task1.java
package seminar1;

public class Task1 {
    public static void main(String[] args) {
        int[] array = {1,2,3,4,5};
        int target = 5;
        System.out.println(findNum(array, target));
    }

    public static boolean findNum(int[] array, int target){
        for (int num : array) {
            if (num == target) {
                return true;
            }
        }
        return false;
    }
}
